<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>empRespStatsDialog</class>
 <widget class="QDialog" name="empRespStatsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButtonListeEmployes_3">
       <property name="text">
        <string>Liste des Employ√©s</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButton_ListeClients_3">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Liste des Clients</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="logo_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>logoProjet_petit.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButton_AfficherChambres_3">
       <property name="text">
        <string>Liste des chambres</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButton_Stats">
       <property name="text">
        <string>Statistiques de l'hotel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="accueil_3">
         <property name="text">
          <string>Accueil</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="account_3">
         <property name="text">
          <string>Compte</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_26">
       <item>
        <widget class="QPushButton" name="pushButton_NbResaYM">
         <property name="text">
          <string>Nombre reservations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_NbClientsVille">
         <property name="text">
          <string>Nombre clients par ville</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_PrixMoyenChambre">
         <property name="text">
          <string>Prix moyen chambre</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_Graphe">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>323</width>
          <height>254</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
